docs/configure-artifact-repository.md

```diff
@@ -75,16 +75,20 @@ artifacts:
 
 ## Configuring AWS S3
 
-Create your bucket and access keys for the bucket. AWS access keys have the same
-permissions as the user they are associated with. In particular, you cannot
-create access keys with reduced scope. If you want to limit the permissions for
-an access key, you will need to create a user with just the permissions you want
-to associate with the access key. Otherwise, you can just create an access key
-using your existing user account.
+First, create your bucket:
```

Comment:
```suggestion
First, create a bucket:
```

- Matching "a policy file" etc below
- Also "a" better implies that it can be _any_ bucket